make singularity gives tar: This does not look like a tar archive

This is on CentOS 7.4.1708.
_git clone git@github.com:EOSC-LOFAR/prefactor-cwl.git
cd prefactor-cwl/
make small
make singularity_
singularity build prefactor.simg docker://kernsuite/prefactor
Docker image path: index.docker.io/kernsuite/prefactor:latest
Cache folder set to /home/hspreeuw/.singularity/docker
Importing: base Singularity environment
Importing: /home/hspreeuw/.singularity/docker/sha256:ae79f251470513c2a0ec750117a81f2d58a50727901ca416efecf297b8a03913.tar.gz
Importing: /home/hspreeuw/.singularity/docker/sha256:c59d01a7e4caf1aba785eb33192fec3f96e4ab01975962bcec10f4989a6edcc6.tar.gz
Importing: /home/hspreeuw/.singularity/docker/sha256:41ba73a9054d231e1f555c40a74762276900cc6487f5c6cf13b89c7606635c67.tar.gz
Importing: /home/hspreeuw/.singularity/docker/sha256:f1bbfd495cc1112b503350686641ee4fa2cea8ccd13fb8a8a302c81dae61d418.tar.gz
Importing: /home/hspreeuw/.singularity/docker/sha256:0c346f7223e24b517358f52c4a3f5f9af1c86e5ddeaee5659c8889846e46d1e2.tar.gz
Importing: /home/hspreeuw/.singularity/docker/sha256:760bbf8251916f26631b53355347bcfdc0274d9b455b2ad9017ddc87b5c5a83c.tar.gz

gzip: /home/hspreeuw/.singularity/docker/sha256:760bbf8251916f26631b53355347bcfdc0274d9b455b2ad9017ddc87b5c5a83c.tar.gz: not in gzip format
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
Cleaning up...
make: *** [prefactor.simg] Error 2

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-.PHONY: clean run small docker
+.PHONY: clean run small
 all: run
 SHELL=bash
 RUN := $(PWD)/runs/run_$(shell date +%F-%H-%M-%S)
@@ -97,10 +97,4 @@ mesos: data/$(SMALL)
 		prefactor.cwl \
 		jobs/job_20sb.yaml | tee $(RUN)/output
 
-docker:
-	docker build docker/ -t kernsuite/prefactor
 
-prefactor.simg:
-	singularity build prefactor.simg docker://kernsuite/prefactor
-
-singularity: prefactor.simg

